### ISSUE TITLE
NTR: Pipecolic acid high in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8532]
New term request Pipecolic acid high in urine (segal_030506185606):   UMLS: 5394149

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000269
- label: Elevated urinary pipecolic acid level
- parents: [Monocarboxylic aciduria (HP:0034653)](https://hpo.jax.org/app/browse/term/HP:0034653)
- pmid: 
- def: The amount of pipecolic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8532
- synonyms:
